packages/itwin/measure-tools/src/tools/MeasureLocationTool.ts

```diff
@@ -122,6 +154,23 @@ MeasureLocationToolModel
     return new MeasureLocationToolModel();
   }
 
+  public override isValidLocation(ev: BeButtonEvent, _isButtonEvent: boolean): boolean {
```

Comment:
Shouldn't there be a check to see if we're targeting a drawing in the first place?
like:
```ts
if (true !== ev.viewport?.view.is2d()) {
  return true;
}
```

Reply:
True, isSheetView seems even more appropriate